spec/resque_spec/matchers_spec.rb

```diff
@@ -455,6 +455,27 @@
       end
     end
 
+    context "with #at_or_in(timestamp, interval)" do
+      let(:interval) { 10 * 60 }
+      let(:scheduled_at) { Time.now + 30 * 60 }
+
+      before(:each) do
+        if [true, false].sample
```

Comment:
Hi @JonaMX,

Thanks for the PR. Using `sample` in a spec is problematic as the spec is not deterministic from run to run. One alternative would be to extract the two examples into a shared group. Then run the shared group with `enqueue_at` and `enqueue_in`.